lib/ecstatic.js

```diff
@@ -57,27 +57,39 @@ var ecstatic = module.exports = function (dir, options) {
     catch (err) {
       return status[400](res, next, { error: err });
     }
-
-    var file = path.normalize(
-          path.join(root,
-            path.relative(
-              path.join('/', baseDir),
-              pathname
+    
+    // try and serve assets
+    var assetsDirectory = '_ecstatic-assets'
```

Comment:
I don't like the shadowing of the directory structure that this does, so it at least has to be possible to turn off. I think the option that turns on icons should also turns on the css---maybe `pretty` is a better name?